includes/size-analysis/upload-cli-android.mdx

```diff
@@ -0,0 +1,16 @@
+1. Configure the [sentry-cli](/cli/)
+
+2. Set up `SENTRY_AUTH_TOKEN` in your environment (you can generate a token [here](https://sentry.sentry.io/settings/auth-tokens/))
```

Comment:
We should prefer to link to the docs on this since there are 4 different ways to provide this token.
https://docs.sentry.io/cli/configuration/#to-authenticate-manually